markets/harmony/index.ts

```diff
@@ -78,6 +79,9 @@ export const HarmonyMarket: IAaveConfiguration = {
       assets: ["USDC", "USDT", "DAI"],
     },
   },
+  RateStrategies: {
+    rateStrategyDebtPrevention,
```

Comment:
?

Reply:
this is the current rate strategy for Harmony to prevent to increase debt in production, as far as I know, should we also reflect this in `markets/` or use the initial one

Reply:
just highlighting because this does not seem to belong to this PR

Reply:
yep can be done in a separate pr

Reply:
ahh no, after checking this is a mandatory change in this PR to comply with Typescript, it was due before we deployed IR checking the ReservesConfig params, but to support the case where there is no assets, IR were moved outside of this config into their own field "RateStrategies".

The previous strategy set was rateStrategyDebtPrevention, so there is no change in harmony markets config, is just adapting it to support the new RateStrategy script logic.

Other markets doesnt need to be changed due they inherit the Aave config which contains the new RateStrategy field.